.github/ISSUE_TEMPLATE/DeprecationTemplate.md

```diff
@@ -0,0 +1,18 @@
+name: Deprecation Notice
+about: A notice for one of few API deprecations
+title: 'Deprecation Notice'
+labels: 'kind/deprecation'
+
+---
+
+**The list of APIs to be deprecated**
+List the APIs with their full namespaces and class
+
+**Describe alternatives for each of the APIs**
+Any alternative APIs can be used to replace each of the decprecated APIs
+
+**Describe the last release version before this deprecation **
+For example: stabilization/2210
+
+**Additional context**
+Add some information about the reasons of the deprecations
```

Comment:
nit: Would it be possible to add a line break after each heading (general markdown linting guidance - could also update to use a heading marker (e.g. `###`) but not critical.